### ISSUE TITLE
Publish the app on F-Droid

### ISSUE THREAD

**Issue by Rudloff** [Techno-Disaster/tldr-flutter#3]
**Is your feature request related to a problem? Please describe.**
tldr-flutter is not available on [F-Droid](https://f-droid.org/) so it can't be easily installed by users that don't have Google Play.

**Describe the solution you'd like**
I intend to submit a merge request to add tldr-flutter to the F-Droid repository.
However, we require that the app developers give their consent before doing so. Would you agree to you app being added to the F-Droid repository?

**Describe alternatives you've considered**
For now, I installed the app with [Aurora Store](https://gitlab.com/AuroraOSS/AuroraStore).

**Additional context**
F-Droid is a FOSS app repository. Every app in the repository is built from source.
(I am an F-Droid contributor.)

Edit: I opened a request for packaging [here](https://gitlab.com/fdroid/rfp/-/issues/1677). (This is were we track the inclusion of new apps.)

**Comment by Techno-Disaster** [comment]
Hey @Rudloff thanks for creating the MR, I haven't found the time to this, but I'm fine with this. I'll also add a comment for the same on gitlab.

**Comment by Techno-Disaster** [comment]
App is now available on f-droid. Thanks for helping out @Rudloff .

[<img src="https://fdroid.gitlab.io/artwork/badge/get-it-on.png" alt="Get it on F-Droid" height="80">](https://f-droid.org/packages/wtf.technodisaster.tldr/)
